docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Set_an_Observation.md

```diff
@@ -0,0 +1,150 @@
+
+2. Configure your job:
+   * **Name ***: Provide a name and an optional description of the job.
+   * **Access**: skip, because Lwm2m test application doesn’t use organizations.
+   * **Target devices**: choose which devices this job will run on.
+   * **Job properties**:
+       * Set **Job type** as `Property`.
+       * Under the **Name** field, select the resource you want to put the observation for and turn the toggle to **True**.
+       * To add parameters to this observation, click **+ Add** and select the respective parameters in the drop-down menu. Specify the attribute in the pop-up window that appears, e.g., `pmin`.
+       * Click **Next**.
+
+    ![Job properties](images-observation/observation-central6.png "Set an Observation and its attributes in Job properties")
+
+    !!! info
+        * **pmin** - the minimum time in seconds between two notifications.
+        * **pmax** - the maximum time in seconds between two notifications. The notification is sent even if the value didn't change.
+
+        Read more about other attributes in our [Brief description of OMA LwM2M](https://avsystem.github.io/Anjay-doc/LwM2M.html#attributes).
+
+3. In **Delivery options**, click **Next**.
+4. In **Schedule**, click **Next**.
+5. Review and click **Run**.
+
+## See value changes
+
+1. In Azure IoT Central, go to **Devices** from the left pane and click on the device you observe. You will see the list of messages that notify about any changes in the desired and reported properties for this device.
+
+2. In the **Raw data** tab, unfold one of the recent messages. In the JSON snippet, you can see that an Observation has been set for a **Telemetry resource ID n6n0n2** (objects, object instances, and resources are separated by `n` in Azure IoT Central).
+
+    ![Raw data](images-observation/observation-central7.png "A message received from the device")
+
+To see the data received about the value changes, scroll right until you find the **Altitude** column. As this is a test example, the value is 0, but in real life, you will see the actual numbers.
+
+![Altitude column](images-observation/observation-central8.png "Any value changes for this resource will be shown in the Altitude column")
+
+!!! Tip
+    If you don’t see any value in the necessary column (e.g., Altitude), scroll all the way right to the very last column **Unmodeled data**. If it contains an entry similar to ``{"n6n0n2":"1634741504000"}``, it means there is a mismatch between Coiote DM and Azure IoT Central templates. Altitude is a Telemetry resource, but apparently it is set as Property in Azure. To fix this error, go to **Device templates** in Azure IoT Central, select the necessary template and find the resource you want to edit. In the **Capability type** column, select `Telemetry` from the drop-down menu, so that it’s consistent with what you have in Coiote DM. Click **Save**. Go back to **Devices** and refresh the page to see the changes.
```

Comment:
Sugeruję zmianę wordingu: necessary > relevant (albo desired, up to you) - w dwóch miejscach tutaj.

Reply:
changed to relevant